gnovm/pkg/gnolang/preprocess.go

```diff
@@ -174,6 +174,10 @@ func initStaticBlocks(store Store, ctx BlockNode, bn BlockNode) {
 			case *ImportDecl:
 				nx := &n.NameExpr
 				nn := nx.Name
+				loc := last.GetLocation()
+				if IsPurePackagePath(loc.PkgPath) && IsRealmPath(n.PkgPath) {
+					panic(fmt.Sprintf("package path %s imports realm path %s", loc.PkgPath, n.PkgPath))
```

Comment:
Maybe it would be good to mention something like "disallowed action: ..." so people know that they are not supposed to do this; the error right now might be a bit unclear. WDYT?

Reply:
we can standardize it to "should not happen, ...", the advantage of this is that other instances of "should not happen" can also be modified in a similar way in the future to enhance readability.

Reply:
wdyt about this [43bc599] ?(https://github.com/gnolang/gno/pull/3042/commits/43bc5995b2341b29465fdfb2b4e1c19922a0fe23)